extension/src/test/runTests.ts

```diff
@@ -198,27 +196,17 @@ async function runTestsForVsCodeVersion(version: string): Promise<void> {
     await runNestedProjectTests(vscodeExecutablePath, tmpDir);
     await runTestsWithoutGradle(vscodeExecutablePath, tmpDir);
   } catch (err) {
-    hasErr = true;
     console.error('Error running tests:', err.message);
-  } finally {
-    await fs.remove(tmpDir);
```

Comment:
there may be a bug between VSCode 1.60.0 (or maybe several versions older) and `test-electron` causing the `user-dir` is occupied after vscode instance exited in windows. Since we run the tests on pipeline, can we just remove the clean process here?

Reply:
what if I want to run tests on my dev machine which is Windows?

Reply:
A temporary folder will be created in `os.tmpdir()` to collect test logs. The size is about 80KB.

Reply:
Will it affect next run? 

BTW, what's the purpose to remove this block? Is there any side-effect for the moment?

Reply:
> Will it affect next run?

No. `mkdtempSync()` generates six random characters to be appended behind a required prefix to create a unique temporary directory. See https://github.com/microsoft/vscode-gradle/blob/d74d483b729e87451bf53696ebe9c3da63503d6f/extension/src/test/runTests.ts#L183

> BTW, what's the purpose to remove this block? Is there any side-effect for the moment?

Due to the bug in https://github.com/microsoft/vscode-gradle/pull/997#discussion_r707044006, the file is occupied, causing this `remove()` throws an error. You can see https://github.com/microsoft/vscode-gradle/runs/3582740912#step:15:382 for the reference.

Reply:
Got it. I'm fine with it. Better open an issue to `test-electron` repo.